build/Versions.props

```diff
@@ -1,7 +1,7 @@
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <VisualStudioVersion>16.3.0</VisualStudioVersion>
-    <ProjectSystemVersion>3.3.0</ProjectSystemVersion>
+    <ProjectSystemVersion>16.3.0</ProjectSystemVersion>
```

Comment:
If they are going to have the same value, can't we just get rid of one entirely?

Reply:
You'll need to update usage throughout the tree, for example: GenerateDependentAssemblyVersionFile.proj.

Reply:
@davkean What is GenerateDependentAssemblyVersionFile.proj? I can't seem to find that anywhere

Reply:
https://github.com/dotnet/project-system/blob/master/build/Scripts/GenerateDependentAssemblyVersionFile.proj

Reply:
What I do when I change/remove a variable is the following:

```
project-system>findstr /sip /c:"VisualStudioVersion" *
```

Reply:
Ah that's very useful. Somehow when I do a search for this file it's not indexed - seems to be true for most files under the /build directory.

Reply:
Another thing to be aware of, is that you also need to search the toolset under:

```
C:\Users\[user]\.nuget\packages\roslyntools.repotoolset
```

To see if it consumes it.

Reply:
roslyntools.repotoolset doesn't consume it